pytket/extensions/quantinuum/backends/api_wrappers.py

```diff
@@ -90,6 +90,7 @@ def __init__(
         use_websocket: bool = True,
         provider: Optional[str] = None,
         support_mfa: bool = True,
+        session: Optional[Session] = None,
```

Comment:
Could you please add this to the documentation?

Reply:
(And maybe to the changelog in `docs/changelog.rst` )

Reply:
done!

Reply:
Thank you!